keyboards/handwired/split_cloud/keymaps/default/keymap.c

```diff
@@ -0,0 +1,246 @@
+    ),
+
+    /* Adjust (Lower + Raise)
+     * ,-----------------------------------------------------------------------------------.
+     * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      | Reset| Debug|      |      |      |      |      |      |      |      |  Del |
+     * |------+------+------+------+------+-------------+------+------+------+------+------|
+     * |      |      |Aud cy|Aud on|AudOff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|Plover|      |
+     * |------+------+------+------+------+------|------+------+------+------+------+------|
+     * |      |Voice-|Voice+|Mus on|MusOff|MidiOn|MidOff|      |      |      |TXBOLT|GEM RP|
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |             |      |      |      |      |      |
+     * `-----------------------------------------------------------------------------------'
+     */
+    [_ADJUST] = LAYOUT_ortho_5x12(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12 ,
+        _______, QK_BOOT, DB_TOGG, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL ,
+        _______, _______, _______, _______, _______, _______, _______, QWERTY,  COLEMAK, DVORAK,  PLOVER,  _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, ST_BOLT, ST_GEM ,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    )
+
+};
+
+// clang-format on
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QWERTY:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_QWERTY);
+            }
+            return false;
+        case COLEMAK:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_COLEMAK);
+            }
+            return false;
+        case DVORAK:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_DVORAK);
+            }
+            return false;
+        case LOWER:
+            if (record->event.pressed) {
+                layer_on(_LOWER);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            } else {
+                layer_off(_LOWER);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+            return false;
+        case RAISE:
+            if (record->event.pressed) {
+                layer_on(_RAISE);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            } else {
+                layer_off(_RAISE);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+            return false;
```

Comment:
These do not need custom keycodes anymore, as this functionality has been added to core: 
https://docs.qmk.fm/#/feature_tri_layer